homeassistant/components/tessie/lock.py

```diff
@@ -81,18 +85,56 @@ def is_locked(self) -> bool | None:
 
     async def async_lock(self, **kwargs: Any) -> None:
         """Enable speed limit with pin."""
+        ir.async_create_issue(
+            self.coordinator.hass,
+            DOMAIN,
+            "deprecated_speed_limit_locked",
+            breaks_in_ha_version="2024.10.0",
+            is_fixable=False,
```

Comment:
All these should be fixable as commented.
Apply `fix_flow` in `strings.json`

Reply:
I have tried, but I dont think I understand this. is_fixable means `True if the issue can be automatically fixed`, which I do not believe this can be fixed automatically since it requires the user to fix automations and scripts. If you have an example llike @joostlek provided, that would be helpful.

Reply:
Set it to True and then in `strings.json` you add something like this (example from `kitchen_sink`)

```json
{
  "issues": {
    "bad_psu": {
      "title": "The power supply is not stable",
      "fix_flow": {
        "step": {
          "confirm": {
            "title": "The power supply needs to be replaced",
            "description": "Press SUBMIT to confirm the power supply has been replaced"
        }
      }
    }
  }
}
```

Reply:
I have copied your provided example and updated the wording.